src/scikit_build_core/settings/auto_cmake_version.py

```diff
@@ -0,0 +1,45 @@
+from __future__ import annotations
+
+import re
+
+from packaging.specifiers import SpecifierSet
+
+__all__ = ["find_min_cmake_version"]
+
+
+def __dir__() -> list[str]:
+    return __all__
+
+
+PATTERN = re.compile(
+    r"""
+    ^                              # Match start of line
+    \s*
+    cmake_minimum_required         # Match the command
+    \s*
+    \(                             # Match opening parenthesis
+    (.*?)                          # Capture any contents (note: fancy strings, etc. should not be present)
```

Comment:
The capture group does not include newlines ([example](https://regex101.com/r/4rF07m/1)). Maybe just capture `([^\)]*)` instead

Reply:
I thought I set DOTALL?

Reply:
Ah right. Regex101 doesn't have that flag to check